src/igvfd/schemas/alignment_file.json

```diff
@@ -76,10 +76,14 @@
         }
     },
     "fuzzy_searchable_fields": [
-        "dbxrefs",
         "@type",
         "content_type",
-        "file_format"
+        "dbxrefs",
+        "file_format",
+        "file_set",
+        "href",
```

Comment:
href and s3_uri are very complex entries to want to search on.  Most of them will have the same beginning 5-15 characters and so in fuzzy search all files will come up with searches like ENCFF.  I think these 2 are poor choices to add to fuzzy search or exact search